docs/detections/invest-guide-run-osquery.asciidoc

```diff
@@ -0,0 +1,51 @@
+[[invest-guide-run-osquery]]
+=== Run Osquery from investigation guides
+Investigations guides can be used with rules and provide guidance for triaging, analyzing, and responding to potential security issues. When you build a custom rule, you can also set up an investigation guide that incorporates Osquery. This allows you to run live queries from a rule's investigation guide as you analyze alerts produced by the rule.
+
+[role="screenshot"]
+image::images/osquery-investigation-guide.png[Shows a live query in an investigation guide]
+
+
+.Requirements
+[sidebar]
+--
+* The {kibana-ref}/manage-osquery-integration.html[Osquery manager integration] must be installed.
+* {agent}'s {fleet-guide}/view-elastic-agent-status.html[status] must be `Healthy`. Refer to {fleet-guide}/fleet-troubleshooting.html[{fleet} Troubleshooting] if it isn't.
+* Your role must have {kibana-ref}/osquery.html[Osquery feature privileges].
+--
+
+[float]
+[[add-live-queries-ig]]
+=== Add live queries to an investigation guide
+
+NOTE: You can only add Osquery to investigation guides for custom rules because prebuilt rules cannot be edited.
+
+Go to Rules page, then do the following:
+
+. Select a rule, then click *Edit rule settings* on the rule details page.
```

Comment:
The lead-in seems like a step in itself, but it could also just bundle them all together:
```suggestion
. Go to *Manage* -> *Rules*, select a rule, then click *Edit rule settings* on the rule details page.
```